.github/workflows/script.sh

```diff
@@ -15,28 +15,21 @@
 
 set -x
 set -e
-export TRAVIS_TAG=${TRAVIS_TAG:-$(git describe --match=v*)}
+export TAG=${TAG:-$(git rev-parse --short "$GITHUB_SHA")}
 
 # TODO(b/171679296): re-enable c++11 support
 #   by downgrading kythe build requirements.
 BAZEL_CXXOPTS=(--cxxopt=-std=c++17)
 
-# Reduce log noise.
-BAZEL_OPTS=(--show_progress_rate_limit=10.0)
-
 case "$MODE" in
-test)
-    bazel test -c opt "${BAZEL_OPTS[@]}" "${BAZEL_CXXOPTS[@]}" //...
-    ;;
-
-compile)
-    bazel build -c opt "${BAZEL_OPTS[@]}" "${BAZEL_CXXOPTS[@]}" //...
+compile-n-test)
+    bazel test --noshow_progress "${BAZEL_CXXOPTS[@]}" //...
+    bazel build -c opt --noshow_progress "${BAZEL_CXXOPTS[@]}" //...
```

Comment:
Both should use `-c opt` consistently so the two commands share the cache.  (Reminder: `CHECK`s in our code are always-on unaffected by `NDEBUG`)

BTW, I'm not opposed running a duplicate job without `-c opt` (or other configuration options) once we get the first one set up.